code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -212,7 +212,7 @@ var/list/blacklisted_pylon_turfs = typecacheof(list(
 				if(L.health != L.maxHealth)
 					new /obj/effect/temp_visual/heal(get_turf(src), "#960000")
 					if(ishuman(L))
-						L.heal_overall_damage(1, 1)
+						L.heal_overall_damage(1, 1, TRUE, 0, 1)
```

Comment:
Replace the second last 0 with FALSE and last 1 with TRUE please.

The parent proc itself should've been refactored to indicate these are meant to be boolean

Reply:
I was a bit confused why they were swapping between true/false and 0/1, kept them consistent with the parameters in case it mattered. Thanks for this!